src/taipy/core/_repository/_filesystem_repository.py

```diff
@@ -228,7 +228,17 @@ def __to_entity(
     def __model_to_entity(self, file_content):
         data = json.loads(file_content, cls=_CustomDecoder)
         model = self.model.from_dict(data)  # type: ignore
-        return self._from_model(model)
+        entity = self._from_model(model)
+        # Add version attribute on old entities. Used to migrate from <=2.0 to >=2.1 version.
+        # To be removed on later versions
```

Comment:
Actually, I think this will stay on 2.2
On Taipy 3.0, we may remove this I guess.